src/hooks/mySql.test.js

```diff
@@ -52,279 +52,338 @@ describe('mySql', () => {
-          .build(),
-      ]);
-      done();
+  describe('v1 -> ', () => {
+    test('hook -> query (text: string, callback: Function) -> success', (done) => {
+      const client = createHookedMySqlV1Client();
+
+      client.query('SELECT * from users', () => {
+        const spans = SpansContainer.getSpans();
+        expect(spans).toEqual([
+          createBaseBuilderFromSpan(spans[0])
+            .withQuery('SELECT * from users')
+            .withConnectionParameters(DUMMY_OPTIONS)
+            .withResponse(createExpectedResponse())
+            .build(),
+        ]);
+        done();
+      });
     });
-  });
 
-  test('v2 -> hook -> query (text: string, callback: Function) -> sqlObject', (done) => {
-    const client = createHookedMySqlV2Client();
-
-    const sql = {
-      _events: {},
-      _eventsCount: 1,
-      _callSite: {},
-      _ended: false,
-      _timer: {
-        _timeout: null,
-      },
-      sql: 'CALL usp_get_pending_requests(?,?,?)',
-      values: ['6784283985', '0', ''],
-      typeCast: true,
-      nestTables: false,
-      _resultSet: null,
-      _results: [],
-      _fields: [],
-      _index: 0,
-      _loadError: null,
-    };
-
-    const sqlObject = { sql };
-
-    const expectedQuery =
-      '{"sql":"CALL usp_get_pending_requests(?,?,?)","values":["6784283985","0",""],"typeCast":true,"nestTables":false}';
-
-    client.query(sqlObject, () => {
-      const spans = SpansContainer.getSpans();
-      expect(spans).toEqual([
-        createBaseBuilderFromSpan(spans[0])
-          .withQuery(expectedQuery)
-          .withConnectionParameters(DUMMY_OPTIONS)
-          .withResponse(createExpectedResponse())
-          .build(),
-      ]);
-      done();
+    test('hook -> query (text: string, values: List, callback: Function) -> success', (done) => {
+      const client = createHookedMySqlV1Client();
+
+      client.query('SELECT * from users', ['123'], () => {
+        const spans = SpansContainer.getSpans();
+        expect(spans).toEqual([
+          createBaseBuilderFromSpan(spans[0])
+            .withQuery('SELECT * from users')
+            .withConnectionParameters(DUMMY_OPTIONS)
+            .withValues(payloadStringify(['123']))
+            .withResponse(createExpectedResponse())
+            .build(),
+        ]);
+        done();
+      });
     });
-  });
 
-  test('v2 -> hook -> query (text: string, callback: Function) -> Unknown query', (done) => {
-    const client = createHookedMySqlV2Client();
-
-    client.query(1, () => {
-      const spans = SpansContainer.getSpans();
-      expect(spans).toEqual([
-        createBaseBuilderFromSpan(spans[0])
-          .withQuery('Unknown')
-          .withConnectionParameters(DUMMY_OPTIONS)
-          .withResponse(createExpectedResponse())
-          .build(),
-      ]);
-      done();
+    test('hook -> query (text: string, callback: Function) -> fail', (done) => {
+      const error = new Error('DuumyError');
```

Comment:
trivial typo: DummyError (copy/pasted 12 times)

Reply:
Fixed in b1086a53a98675ac5ccae59c5c31b5303d8d404d